python-packages/contract_wrappers/setup.py

```diff
@@ -128,11 +105,27 @@ def run(self):
         rmtree(".mypy_cache", ignore_errors=True)
         rmtree(".tox", ignore_errors=True)
         rmtree(".pytest_cache", ignore_errors=True)
-        rmtree("src/0x_contract_wrappers.egg-info", ignore_errors=True)
+        rmtree(
+            path.join("src", "0x_contract_wrappers.egg-info"),
+            ignore_errors=True,
+        )
         # generated files:
-        for contract in CONTRACTS_TO_BE_WRAPPED:
+        print("Removing src/zero_ex/contract_wrappers/*/__init__.py...")
+        for contract in glob(
+            path.join(
+                ".", "src", "zero_ex", "contract_wrappers", "*", "__init__.py"
```

Comment:
small nit here - this path is relative to the directory the current process is in, and not relative to the directory the current file is in. I think you want the latter?

**Example**

If I ran..
```
cd /tmp/
python path/to/monorepo/python-packages/contract_wrappers/setup.py
```

LOC 114 would result in `/tmp/src/zero_ex/contract_wrappers/*/__init__.py` while I think you wanted it to be `/tmp/path/to/monorepo/python-packages/contract_wrappers/src/zero_ex/contract_wrappers/*/__init__.py`

Reply:
You're absolutely right on this... but I don't think it's worth accommodating at this point.  The script already generally lacks support for this type of interaction.

Reply:
yup yup I agree